Index: Project/src/GUI.java
===================================================================
diff --git a/Project/src/GUI.java b/Project/src/GUI.java
deleted file mode 100644
--- a/Project/src/GUI.java	(revision 8339a2fabe8d5f40ede7743755d4ac38461a601e)
+++ /dev/null	(revision 8339a2fabe8d5f40ede7743755d4ac38461a601e)
@@ -1,24 +0,0 @@
-import javax.swing.*;
-import java.awt.*;
-
-public class GUI{
-    public GUI(){
-        JFrame frame = new JFrame();
-
-        JPanel panel = new JPanel();
-        panel.setBorder(BorderFactory.createEmptyBorder(300, 300, 10, 10));
-        panel.setLayout(new GridLayout(0,1));
-
-        frame.add(panel, BorderLayout.CENTER);
-        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
-        frame.setTitle("Project");
-        frame.pack();
-        frame.setVisible(true);
-
-    }
-
-    public static void main(String[] args){
-        new GUI();
-    }
-
-}
Index: README.md
===================================================================
diff --git a/README.md b/README.md
deleted file mode 100644
--- a/README.md	(revision 8339a2fabe8d5f40ede7743755d4ac38461a601e)
+++ /dev/null	(revision 8339a2fabe8d5f40ede7743755d4ac38461a601e)
@@ -1,7 +0,0 @@
-Grupo11:
- -> 92657 - Miguel Cleto Barão Rato;
- -> 92689 - Pedro Afonso Aresta Morais Teixeira Santana;
- -> 93019 - Filipe Alexandre Sequeira Gonçalves;
- ->
-
-
